package.json

```diff
@@ -20,13 +27,24 @@
   "bugs": {
     "url": "https://github.com/surikaterna/kuery/issues"
   },
+  "prettier": "prettier-config-surikaterna",
   "devDependencies": {
-    "istanbul": "^0.3.2",
+    "@swc/cli": "^0.1.57",
+    "@swc/core": "^1.2.220",
+    "@swc/jest": "^0.2.22",
+    "@types/jest": "^28.1.6",
```

Comment:
I don't think we need this, since `jest` provide their own types. Try removing this and verify that it still works.

Reply:
Is needed.